app/models/export/relation.rb

```diff
@@ -13,10 +13,10 @@ def run
         return
       end
 
-      log "Exporting #{model_plural} to #{filename}"
-      File.new(filename, 'w+').tap do |file|
+      log "Exporting #{model_plural}"
+      Tempfile.new([model_plural, '.csv']).tap do |file|
         export_class.new(@relation, file).run
-        file.close
+        file.rewind
```

Comment:
Out of interest, what's the advantage of calling rewind here?

Reply:
Without calling `rewind` first, calling `read` would return a blank string.

Reply:
OK, that's definitely fair.